<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Menu</class>
 <widget class="QWidget" name="Menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="selectlabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>请选择词库：</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="cet4button">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>115</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>英语四级</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="GREbutton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>150</y>
     <width>115</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>GRE</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="numlineEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>210</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="numlabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>请输入本次练习数量：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startbutton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>280</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>开始练习！</string>
   </property>
  </widget>
  <widget class="QPushButton" name="endbutton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>380</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLabel" name="userlabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>请输入用户名：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="namelineEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="newbutton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>70</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>新建。。。</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
